provisioner/ibm-s3fs-provisioner.go

```diff
@@ -409,44 +509,72 @@ func (p *IBMS3fsProvisioner) Provision(options controller.VolumeOptions) (*v1.Pe
 		}
 
 		if creds.APIKey != "" && creds.ServiceInstanceID == "" {
-			return nil, errors.New(pvcName + ":" + clusterID + ":cannot create bucket using API key without service-instance-id")
+			return nil, errors.New(pvcName + ":" + clusterID + " :cannot create bucket using API key without service-instance-id")
 		}
+
+		contextLogger.Info(pvcName + ":" + clusterID + " :creating bucket: " + pvc.Bucket)
 		msg, err = sess.CreateBucket(pvc.Bucket)
 		if msg != "" {
-			contextLogger.Info(pvcName + ":" + clusterID + ":" + msg)
+			contextLogger.Info(pvcName + ":" + clusterID + " : " + msg)
 		}
 		// When using existing bucket with auto-create-bucket: true
 		if err != nil {
 			if strings.Contains(fmt.Sprintf("%v", err), "BucketAlreadyExists") {
 				err1 := sess.CheckBucketAccess(pvc.Bucket)
 				if err1 != nil {
-					return nil, fmt.Errorf(pvcName+":"+clusterID+":cannot create bucket %s: %v", pvc.Bucket, err)
+					return nil, fmt.Errorf(pvcName+":"+clusterID+" :cannot create bucket %s: %v", pvc.Bucket, err)
 				}
 				valBucket = false
```

Comment:
So at line no. 560, for `existing bucket with different name`, we are checking bucket access after setting access policy.
So, on similar line, do we need to check bucket access after setting bucket policy for `existing bucket with same name`?

Reply:
As per discussion, removed the `err1 := sess.CheckBucketAccess(pvc.Bucket)` as it might throw error if user is trying to use an existing bucket for which AccessPolicy is already set. (throws Not Authorized).
To fix that scenario bucket access will be checked after the access policy is set for bucket.